_pages/user-experience/faq-content.md

```diff
@@ -15,9 +15,60 @@ sidenav:
     href: "user-experience/failure-proof/"
   - text: Knowledge articles
     href: "user-experience/knowledge-articles/"
-  - text: FAQ content 
+  - text: FAQ content
     href: "user-experience/faq-content/"
 
 ---
 
-## FAQ Content
+## Add FAQ content to inform users about Login.gov (Optional)
+
+When you first integrate with Login.gov, we recommend adding frequently asked questions (FAQ) content to ensure that users are able to understand that your application uses Login.gov as a sign-in service.. The next few sections outline plain language for agencies to leverage when creating that content.
```

Comment:
extra period after the first sentence

Reply:
fixed in https://github.com/18F/identity-dev-docs/pull/416/commits/a54ff2f891831b7429c17240343d2d83cc41b378